scripts/overrides/index.js

```diff
@@ -67,6 +67,19 @@ function overrides(file, schema) {
           `and then update \`SUPPORTED_GHES_VERSION\`.`;
   }  
 
+
+  // Fix for the `GET /repos/{owner}/{repo}/contents/{path}` endpoint.
+  // This is so we can discriminate between the different response types.
+  // ref: https://github.com/github/rest-api-description/issues/165
+  // ref: https://github.com/octokit/rest.js/issues/32
+  const defenitions = schema.components.schemas;
```

Comment:
This line is giving problems with our CI: https://github.com/octokit/openapi/runs/7258962452?check_suite_focus=true

Reply:
also typo, should be `definitions`